allennlp/nn/util.py

```diff
@@ -113,6 +114,10 @@ def arrays_to_variables(data_structure: Dict[str, Union[dict, numpy.ndarray]],
         using this function. This is useful during inference for passing
         tensors representing a single example to a Pytorch model which
         would otherwise not have a batch dimension.
+    for_training : ``bool``, optional (default = ``True``)
```

Comment:
Also I think it would be more accurate to say "this makes inference slightly faster and considerably more memory efficient".